doc/.wordlist.txt

```diff
@@ -1,38 +1,49 @@
-
+addons
 API
 APIs
-CLI
+balancer
 Charmhub
+CLI
+crypto
+cryptographic
+cryptographically
+customization
```

Comment:
```suggestion
customisation
```
Could we use 'customisation', please, to keep up the "BritEng by default" pretense?

Reply:
At first I used 'customization' because the key in the configuration is written with a `z`. But it is a matter of choosing with what we want to be coherent and I have no strong opinion about it so 'customisation' is fine.

Reply:
>  the key in the configuration is written with a z

Which would be perfectly reasonable, but Canonical wants to insist on BritEng for docs, so we're essentially documenting our AmEng software in a different language 🤦‍